<ui version="4.0" >
 <class>CameraSettingClass</class>
 <widget class="QDialog" name="CameraSettingClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Camera Info</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>3</x>
     <y>11</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Eye:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="eyeX" >
   <property name="geometry" >
    <rect>
     <x>69</x>
     <y>11</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="eyeY" >
   <property name="geometry" >
    <rect>
     <x>177</x>
     <y>11</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="eyeZ" >
   <property name="geometry" >
    <rect>
     <x>285</x>
     <y>11</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dirY" >
   <property name="geometry" >
    <rect>
     <x>177</x>
     <y>41</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>3</x>
     <y>41</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Direction:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dirZ" >
   <property name="geometry" >
    <rect>
     <x>285</x>
     <y>41</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dirX" >
   <property name="geometry" >
    <rect>
     <x>69</x>
     <y>41</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="upZ" >
   <property name="geometry" >
    <rect>
     <x>285</x>
     <y>70</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>true</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>4</x>
     <y>70</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Up:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="upY" >
   <property name="geometry" >
    <rect>
     <x>177</x>
     <y>70</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="upX" >
   <property name="geometry" >
    <rect>
     <x>69</x>
     <y>70</y>
     <width>102</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="buttonSymbols" >
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="correctionMode" >
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals" >
    <number>3</number>
   </property>
   <property name="minimum" >
    <double>-1000000000.000000000000000</double>
   </property>
   <property name="maximum" >
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep" >
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
